Parse/TPTP.hpp

```diff
@@ -38,6 +39,7 @@ namespace Kernel {
 };
 
 namespace Parse {
+  using namespace Kernel;
```

Comment:
I think this then means that you can use anything from `Kernel` as if it were in `Parse`, like `Parse::Clause` - was this the intention?

Reply:
I just had a look again and before i added this we just had a `using namespace Kernel` in global scope. I thought this was a bit better because we don't leak into global scope for other files including `TPTP.hpp`. But I don't have any strong opiion on that. It was just accidentally broken when i merged things, so i fixed it again like this. I can also do the global import instead. as you like :)

Reply:
Ugh, OK - this is better than that! I'll go around and do another `using namespace`-in-header file purge at some point. :-)